health/healthcheck.go

```diff
@@ -0,0 +1,58 @@
+package health
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Financial-Times/draft-annotations-api/annotations"
+	health "github.com/Financial-Times/go-fthealth/v1_1"
```

Comment:
I'm surprised this works, and it's downright confusing. Maybe alias this package to `fthealth` for clarity?